recipes/dye-score/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "dye-score" %}
+{% set version = "0.1.0" %}
+{% set hash_type = "sha256" %}
+{% set hash = "0c50f781b5f65845840dfe86b69c601f81ae723ea63aed57d32e8ec4994b62e88" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name|replace("-", "_") }}/{{ name|replace("-", "_") }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - setuptools_scm
+    - pytest >=3.8.2
+
+  run:
+    - python
+    - bokeh
+    - dask
+    - numpy
+    - pyarrow
+    - pyspark
+    - pyyaml
+    - xarray
+    - zarr
+
+test:
+  imports:
+    - dye_score
+
+about:
+  home: https://pypi.python.org/pypi/dye-score
+  license: MPL 2.0
```

From https://opensource.org/licenses/alphabetical:
```suggestion
  license: MPL-2.0
```